src/Schema/SchemaValidator.php

```diff
@@ -126,7 +126,7 @@ public function validate($data, CebeSchema $schema, ?BreadCrumb $breadCrumb = nu
             }
 
             if (isset($schema->properties) && count($schema->properties)) {
-                $additionalProperties = $schema->additionalProperties ?? null;
+                $additionalProperties = $schema->additionalProperties ?? null; // defaults to true
```

Comment:
Having this comment here means it's something wrong with typehints or default value. Maybe just have `?? true`? Or it won't work?

Reply:
Just concerning about not self-documentary code, ok